tests/hardware_tests/configs/config.yaml

```diff
@@ -1,8 +1,20 @@
 WEIGHTS:
-  NAV: "weights/CUTOUT_WT_True_SD_200_ckpt.99.pvp.pth"
-  GAZE: "weights/final_paper/gaze_normal_32_seed100_1649708902_ckpt.38.pth"
-  PLACE: "weights/final_paper/place_10deg_32_seed300_1649709235_ckpt.75.pth"
+  # Regular Nav
+  NAV: "weights/torchscript/CUTOUT_WT_True_SD_200_ckpt.99.pvp_combined_net.torchscript" #"weights/CUTOUT_WT_True_SD_200_ckpt.99.pvp.pth"
+  
+  # Static gaze
+  GAZE: "weights/torchscript/gaze_normal_32_seed100_1649708902_ckpt.38_combined_net.torchscript" #"weights/final_paper/gaze_normal_32_seed100_1649708902_ckpt.38.pth"
+
```

Comment:
let's remove the older weights here?

Reply:
Removed.
@tusharsangam we have removed the old weights from both regular and test configs.yaml (Just fyi)